antidote-v1.0/MP3-curriculum-images.md

```diff
@@ -0,0 +1,4 @@
+TBD
+
+<< Document to discuss process for automatically building curriculum images from scratch, including the building of custom kernels >>
+
```

Comment:
My existing, **very rough** notes below:

```suggestion
Labtainers?
https://twitter.com/katacontainers
labtainers?


https://www.weave.works/blog/fire-up-your-vms-with-weave-ignite
https://felixwiedmann.de/ignite/

https://blog.openshift.com/re-imagining-virtualization-with-kubernetes-and-kubevirt/

https://medium.com/@alezzandro/kubernetes-and-virtualization-kubevirt-will-let-you-spawn-virtual-machine-on-your-cluster-e809914cc783

Could we provide a packer like experience for building a vm-in-docker process?
Is this what kubevirt does?

Legal Hurdles
https://kubevirt.io/
Kata containers

Better container security
More flexible/controlled networkpolicies (i.e. maybe allow traffic to certain things based on lesson)?
Obergix mentioned some additional security to prevent things like cryptomining in the container
https://medium.com/@gokulchandrapr/kata-containers-on-kubernetes-and-kata-firecracker-vmm-support-28abb3a196e7
https://www-public.imtbs-tsp.eu/~berger_o/weblog/2019/04/26/testing-kubevirt-for-running-vms-inside-kubernetes-in-a-vagrant-qemu-vm/
```